task_planning/scripts/mushr_bag_file.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+import rosbag
+import rospkg
+import numpy as np
+# import bagpy
+# from bagpy import bagreader
+
+base_path = rospkg.RosPack().get_path('task_planning') + '/data/'
```

Comment:
I would prefer to have bags _outside_ the repo. (i.e. at the same level of `build/` and `devel/`. If you want the files here, could we it parameterized? (and probably add to `.gitignore`)

Reply:
It is parameterized right now (check `single_shot.launch`). I have added it to the `.gitignore`. But if you strongly prefer having it outside the repo, lmk (need to check how to get the workspace path inside roslaunch)